src/main/java/frc/robot/Robot.java

```diff
@@ -26,11 +27,14 @@
  */
 public class Robot extends LoggedRobot {
     private RobotContainer robotContainer;
+    public SimIO simIO = new SimIO();
```

Comment:
We would like the EndgameSubsystem to be in RobotContainer - that would be a better place to init SimIO